### ISSUE TITLE
Col_widths and colsVisibility

### ISSUE THREAD

**Issue by cjohnsonuk** [koalyptus/TableFilter#507]
I've not noticed this before so I'm not sure if this is new since 0.5.33 or if I've just never used both these features in combination.  

Column widths appear to apply to the visible columns, not to the actual columns position.  Eg if I have a 2 column table with: 
col_widths: ["10px","100px"]
eg an ID and a description when both are visible the column widths are 10px for column 0 (the ID) and 100px for column 1 the description.

When hiding column 0 the width of the description becomes 10px

eg open http://koalyptus.github.io/TableFilter/columns-visibility.html 

and run this code 
```
tf.destroy();
   var tfConfig = {
        base_path: 'tablefilter/',
        paging: true,
        state: { types: ['cookie'] },
        alternate_rows: true,
        rows_counter: true,
        btn_reset: true,
        btn_reset_text: 'Clear',
        status_bar: true,
		col_widths: ["50px","100px","200px","50px","100px","300px"],
        col_1: 'select',
        col_2: 'select',
        col_types: [
            'number', 'string', 'string',
            'formatted-number', 'formatted-number', 'string'
        ],

        /* columns visibility and sort extension */
        extensions: [{
              name: 'colsVisibility',
              at_start: [3, 5],
              text: 'Columns: ',
              enable_tick_all: true
          }, {
              name: 'sort'
        }]
    };
    var tf = new TableFilter('demo', tfConfig);
    tf.init();

```
(this is the existing config copied from the page with col_widths parameters added).

As you hide columns on the left of the table the columns move left and adopt the width of the column previously occupying that space rather than keeping their own width.

We're on v 0.5.33 and it appears on the live examples as well (v0.5.40)

Sorry for finding these "opportunities for improvement" ;-)

**Comment by koalyptus** [comment]
Tx for pointing this out, after a quick investigation seems to be a unexpected behaviour, there is always room for improvement 😄

**Comment by koalyptus** [comment]
@cjohnsonuk should now be fixed starting from `v0.5.42`
